src/Calculator/Views/UnitConverter.xaml.h

```diff
@@ -40,6 +40,14 @@ namespace CalculatorApp
             }
         }
 
+        property Windows::UI::Xaml::HorizontalAlignment FlowDirectionHorizontalAlignment
```

Comment:
Can this just be
```suggestion
        PROPERTY_R(Windows::UI::Xaml::HorizontalAlignment, FlowDirectionHorizontalAlignment);
```
?

Reply:
ping @rudyhuyn

Reply:
Yes it can, I will update this PR tonight